pkg/sif/descriptor.go

```diff
@@ -295,6 +300,20 @@ func (d Descriptor) SBOMMetadata() (SBOMFormat, error) {
 	return s.Format, nil
 }
 
+// OCIBlobMetadata returns the media for a OCI blob object.
+func (d Descriptor) OCIBlobMetadata() (string, error) {
+	if got, want := d.raw.DataType, DataOCIBlob; got != want {
+		return "", &unexpectedDataTypeError{got, []DataType{want}}
```

Comment:
Same here, probably want to support this for `DataOCIRootIndex`?
```suggestion
	if got := d.raw.DataType; got != DataOCIRootIndex && got != DataOCIBlob {
		return "", &unexpectedDataTypeError{got, []DataType{DataOCIRootIndex, DataOCIBlob}}
```